workflow-varscan/README.md

```diff
@@ -27,37 +28,35 @@ java -jar cromwell.jar run varscan.wdl --inputs inputs.json
 #### Required workflow parameters:
 Parameter|Value|Description
 ---|---|---
-`inputTumor`|File|tumor file
-`inputNormal`|File|normal file
-`inputTumorIndex`|File|.bai index for tumor file
-`inputNormalIndex`|File|.bai index for normal file
+`inputTumor`|File|input .bam file for tumor sample
+`inputNormal`|File|input .bam file for normal sample
+`inputTumorIndex`|File|input .bai file for tumor sample
+`inputNormalIndex`|File|input .bai file for normal sample
 
 
 #### Optional workflow parameters:
 Parameter|Value|Default|Description
 ---|---|---|---
-`outputFileNamePrefix`|String|""|Optional prefix for result files
-`bedIntervalsPath`|String|""|Optional path to a bed file with intervals
-`chromRegions`|Array[String]|["chr1:1-249250621", "chr2:1-243199373", "chr3:1-198022430", "chr4:1-191154276", "chr5:1-180915260", "chr6:1-171115067", "chr7:1-159138663", "chr8:1-146364022", "chr9:1-141213431", "chr10:1-135534747", "chr11:1-135006516", "chr12:1-133851895", "chr13:1-115169878", "chr14:1-107349540", "chr15:1-102531392", "chr16:1-90354753", "chr17:1-81195210", "chr18:1-78077248", "chr19:1-59128983", "chr20:1-63025520", "chr21:1-48129895", "chr22:1-51304566", "chrX:1-155270560", "chrY:1-59373566", "chrM:1-16571"]|Comma-delimited list of chromosomal regions, by default canonical chromosomes for hg19
+`outputFileNamePrefix`|String|""|Output file(s) prefix
+`bedIntervalsPath`|String|""|Path to a .bed file used for targeted variant calling
+`chromRegions`|Array[String]|["chr1:1-249250621", "chr2:1-243199373", "chr3:1-198022430", "chr4:1-191154276", "chr5:1-180915260", "chr6:1-171115067", "chr7:1-159138663", "chr8:1-146364022", "chr9:1-141213431", "chr10:1-135534747", "chr11:1-135006516", "chr12:1-133851895", "chr13:1-115169878", "chr14:1-107349540", "chr15:1-102531392", "chr16:1-90354753", "chr17:1-81195210", "chr18:1-78077248", "chr19:1-59128983", "chr20:1-63025520", "chr21:1-48129895", "chr22:1-51304566", "chrX:1-155270560", "chrY:1-59373566", "chrM:1-16571"]|Regions used for scattering tasks, need to be assembly-specific
```

Comment:
Shesmu does not provided lists in sorted order, but this list needs to be in sorted order for vcf-concat. For now, maybe this should be a string (like in bmpp: https://github.com/oicr-gsi/bam-merge-preprocessing/blob/master/bamMergePreprocessing.wdl#L24).

Reply:
And the accompanying split function to get it back into an ordered Array[String]: https://github.com/oicr-gsi/mutect2/blob/master/mutect2.wdl#L97